frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/repository/RepositoryGraphBuilder.java

```diff
@@ -1,9 +1,9 @@
 package com.virtuslab.gitmachete.frontend.graph.repository;
 
 import java.util.ArrayList;
-import java.util.List;
 import java.util.Optional;
 
+import io.vavr.collection.List;
```

Comment:
**line 36:** `public static final IBranchGetCommitsStrategy EMPTY_GET_COMMITS = __ -> io.vavr.collection.List.empty();`

We want just `List.empty()` here, don't we?

Reply:
Fixed on #182